docs/user_guide/getting-started.md

```diff
@@ -0,0 +1,66 @@
+# Quick Start
+
+Set up Cinema 4D and AWS Deadline Cloud in just a few steps.
+
+## What You'll Need
+
+- **Cinema 4D 2024 or 2025** installed on your workstation
+- **AWS Account** with Deadline Cloud access
+- **Windows or macOS** workstation for job submission
+- **Deadline Cloud monitor** ([download here](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/monitor-onboarding.html))
+- **Access to an AWS Deadline Cloud farm** with either:
+    - A Windows service-managed fleet, or
+    - A customer-managed fleet with Cinema 4D, the Cinema 4D adaptor, and licensing set up
+
+## Step 1: Install the Submitter (5 minutes)
+
+The submitter adds AWS Deadline Cloud functionality to Cinema 4D's Extensions menu, allowing you to submit your scene directly to Deadline Cloud to manage the rendering.
+
+**[Download the Official Installer](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/submitter.html)** ← Start here (recommended)
+
+1. Run the installer and follow the on-screen instructions
+2. Launch Cinema 4D after installation
+3. Verify the submitter appears in `Extensions` > `AWS Deadline Cloud Submitter`
+
+## Updating the Submitter
+
+To update the submitter to the latest version, download and run the latest submitter installer.
+
+## Step 2: Submit Your First Render (2 minutes)
+
+1. Open Cinema 4D and load a test scene. **Or use our sample scenes:** Go to [`test_scenes`](https://github.com/aws-deadline/deadline-cloud-for-cinema-4d/tree/mainline/test/integ/test_scenes), select a scene folder, navigate to its `scene` subfolder, and run `scene.py` in Cinema 4D via `Extensions` > `User Scripts` > `Run Script` (this will generate a .c4d file for you)
```

Comment:
Nit: I think we should out the test scene guidance. The user guide is more geared towards end users using the GUI instead of developers using code. Users will probably also have a scene already.